cognite/client/_api/ai/tools/documents.py

```diff
@@ -0,0 +1,114 @@
+from __future__ import annotations
+
+from collections.abc import Sequence
+
+from cognite.client._api_client import APIClient
+from cognite.client.data_classes.ai import Answer, AnswerLanguage, Summary
+from cognite.client.data_classes.data_modeling import NodeId
+from cognite.client.utils.useful_types import SequenceNotStr
+
+
+class AIDocumentsAPI(APIClient):
+    _RESOURCE_PATH = "/ai/tools/documents"
+
+    def summarize(
+        self,
+        ids: Sequence[int] | None = None,
+        external_ids: SequenceNotStr[str] | None = None,
+        instance_ids: Sequence[NodeId] | None = None,
```

Comment:
If you use "Identifier trick" as mentioned above, supporting single inputs will automatically be supported. I suggest using parameter names in singular case:
```suggestion
        id: int | Sequence[int] | None = None,
        external_id: str | Sequence[str] | None = None,
        instance_id: NodeId | Sequence[NodeId] | None = None,
```